.github/workflows/build-git-annex-debianstandalone.yaml

```diff
@@ -124,6 +124,17 @@ jobs:
         run: |
           sudo dpkg -i git-annex*.deb
 
+      - name: Set up SSH target
+        shell: bash
+        # TODO: Drop the release condition once 0.13.1 is released.
+        run: |
+          if [ "${{ matrix.version }}" != "release" ]; then
+            curl -fSsL \
+              https://raw.githubusercontent.com/datalad/datalad/master/tools/ci/prep-travis-forssh.sh \
+              | bash;
+            echo "::set-env name=DATALAD_TESTS_SSH::1";
```

Comment:
is that some github actions specific way to export env vars from one step into another and regular `export DATALAD_TESTS_SSH=1` wouldn't work?

Reply:
I tried `export DATALAD_TESTS_SSH=1` at first and the SSH tests were skipped, so it seems like those values don't persist in subsequent steps.

The snippet above was the first thing I [stumbled on][0] that worked, though there might be better ways to do it.

[0]: https://docs.github.com/en/actions/reference/workflow-commands-for-github-actions#setting-an-environment-variable

Once the released version of DataLad has the SSH test changes, I think we can just set it via `env`, though.